mlrun/runtimes/base.py

```diff
@@ -116,7 +116,6 @@ def __init__(
         disable_auto_mount=False,
         clone_target_dir=None,
     ):
```

Comment:
only white-space changes in this file, is this created by some linter etc? If not, please remove

Reply:
linter...